x/posts/internal/simulation/utils.go

```diff
@@ -125,14 +125,16 @@ func RandomHashtag(r *rand.Rand) string {
 }
 
 // RandomMedias returns a randomly generated list of post medias
-func RandomMedias(r *rand.Rand) types.PostMedias {
+func RandomMedias(r *rand.Rand, accs []sim.Account) types.PostMedias {
 	mediaNumber := r.Intn(20)
 
+	acc, _ := sim.RandomAcc(r, accs)
+
 	postMedias := make(types.PostMedias, mediaNumber)
 	for i := 0; i < mediaNumber; i++ {
 		host := RandomHosts[r.Intn(len(RandomHosts))]
 		mimeType := RandomMimeTypes[r.Intn(len(RandomMimeTypes))]
-		postMedias[i] = types.NewPostMedia(host+strconv.Itoa(i), mimeType)
+		postMedias[i] = types.NewPostMedia(host+strconv.Itoa(i), mimeType, []sdk.AccAddress{acc.Address})
```

Comment:
I think we should probably randomize even the number of tags to be used here. So: 

1. Generate a random int `x`
2. Get `x` accounts
3. Tag each account into the media